packages/jupyter-ai/jupyter_ai/extension.py

```diff
@@ -95,6 +95,15 @@ class AiExtension(ExtensionApp):
         config=True,
     )
 
+    generation_log_dir = Unicode(
```

Comment:
Would it be useful to generalize this to `error_logs_dir`, which can be used to log all errors, for example errors found during /learn where some files might fail to load/index, while others are ok?

Reply:
Thanks, I changed the name in 0f6d17507b320b01f5d2f4de6c70df797acc57c